package.json

```diff
@@ -64,7 +64,7 @@
     "@storybook/cli": "^5.3.8",
     "@storybook/polymer": "^5.3.8",
     "@storybook/storybook-deployer": "^2.8.1",
-    "@wdio/browserstack-service": "^7.0.0",
+    "@wdio/browserstack-service": "^6.1.12",
```

Comment:
@alazzara I had to revert this version bump that I just did the other day because I couldn't get baselines to run at all locally!